src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -245,9 +245,14 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
     margin: 0;
   }
 
-  // If item is used within a group, add margin back
-  .pf-c-data-toolbar__group .pf-c-data-toolbar__item {
-    margin-right: var(--pf-c-data-toolbar--spacer);
+  .pf-c-data-toolbar__group {
+    display: grid;
+    grid-row-gap: var(--pf-c-data-toolbar__expandable-content--m-expanded--GridRowGap);
```

Comment:
Trying to catch up on the issue so I may be incorrect, but should we be using this variable here for the `grid-row-gap` property? Looks like the class that its being applied to is `pf-c-data-toolbar__group` rather than `expandable content`

Reply:
@christiemolloy `.pf-c-data-toolbar__group`'s grid row gap, in this case, should always match that of `--pf-c-data-toolbar__expandable-content--m-expanded--GridRowGap`. We're passing the parent's layout to any descendant `__group` so that `.pf-c-data-toolbar__content` and `.pf-c-data-toolbar__expandable-content`'s structure can match. Expandable content is a single column grid when this applies.